sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionTestEnvironment.cs

```diff
@@ -16,5 +18,27 @@ public class MonitorIngestionTestEnvironment : TestEnvironment
         public string StreamName => GetRecordedVariable("INGESTION_STREAM_NAME");
 
         public string TableName => GetRecordedVariable("INGESTION_TABLE_NAME");
+
+        public IngestionAudience GetAudience()
+        {
+            Uri authorityHost = new(AuthorityHostUrl);
+
+            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
+            {
+                return IngestionAudience.AzurePublicCloud;
+            }
+
+            if (authorityHost == AzureAuthorityHosts.AzureChina)
+            {
+                return IngestionAudience.AzureChina;
+            }
+
+            if (authorityHost == AzureAuthorityHosts.AzureGovernment)
+            {
+                return IngestionAudience.AzureGovernment;
+            }
+
+            throw new NotSupportedException($"Cloud for authority host {authorityHost} is not supported.");
```

Comment:
A [C# switch expression](https://learn.microsoft.com/dotnet/csharp/language-reference/operators/switch-expression) would be a nice way to clean up this logic.

Reply:
Leaving this as-is - following the pattern from TextAnalytics :)